src/tests/end-to-end/tests/details-view/headings.test.ts

```diff
@@ -73,7 +73,7 @@ describe('Details View -> Assessment -> Headings', () => {
                     headingsPage,
                     detailsViewSelectors.mainContent,
                 );
-                expect(results).toHaveLength(0);
+                expect(results).toHaveLength(1);
```

Comment:
can we leave a comment in the file linking to the issue?

Reply:
Added